tests/integration/client/feedback/training/test_trainer.py

```diff
@@ -107,3 +103,33 @@ def test_prepare_for_training_text_classification(
 
     if Path(__OUTPUT_DIR__).exists():
         shutil.rmtree(__OUTPUT_DIR__)
+
+
+@pytest.mark.parametrize(
+    "callable",
+    (
+        lambda: TrainingTaskMapping.for_text_classification(None, None),
+        lambda: TrainingTaskMapping.for_direct_preference_optimization(None),
+        lambda: TrainingTaskMapping.for_reward_modelling(None),
+        lambda: TrainingTaskMapping.for_supervised_fine_tuning(None),
+    ),
+)
+def test_deprecations(callable) -> None:
```

Comment:
```suggestion
def test_deprecations(callable: Callable[..., Any]) -> None:
```

Reply:
Besides that you probably also need to include `from typing import Callable`

Reply:
e677447831dc9d7dd53abff2bdfed7934b237a40